### ISSUE TITLE
Travis fails on `compiler-test`

### ISSUE THREAD

**Issue by asr** [agda/agda#3239]
When running the `compiler-test` on GHC 8.0.2 Travis is failing with the error:

"No output has been received in the last 10m0s, this potentially indicates a stalled build or something wrong with the build itself."

See the jobs [432305585](https://travis-ci.org/agda/agda/jobs/432305585), [432413810](https://travis-ci.org/agda/agda/jobs/432413810), and [432492508](https://travis-ci.org/agda/agda/jobs/432492508).

**Comment by asr** [comment]
The  `travis_wait` command didn't fix the issue.

**Comment by asr** [comment]
Current status:

The `compiler-test` started to fail *only on GHC 8.0.2* on Travis with the error

> "No output has been received in the last 10m0s...

after commit ec3e855918e34ab27d94443c3de77c97b8d8024b. This commit updated the version of the standard library used  by the Agda repository. This update includes approx 70 commits from the standard library.

I have done the following tests (in personal branches):

1. I increased the timeout in Travis for the `compiler-test` from 10 min to 30 min and the test continue failing. The test passed using a timeout of 40 min, but then Travis fails due to the total timeout for the job.

2. I reverted the above commit and it fixed the problem.

3. I couldn't reproduce the issue using GHC 8.0.2. The `compiler-test` need approx 8 min in my machine: 

```bash
$ export PARALLEL_TESTS=2
$ time make AGDA_TESTS_OPTIONS="-j${PARALLEL_TESTS} --hide-successes" compiler-test

======================================================================
========================== Compiler tests ============================
======================================================================
all
  Compiler
    MAlonzo
      simple
        Issue2524:             DISABLED (0.00s)
        Sharing:               DISABLED (0.00s)

Ignored 2 disabled tests.
All 86 tests passed (346.26s)

real	5m46.568s
user	7m27.570s
sys	0m19.007s
```

I have no idea why the `compiler-test` is failing *only on GHC 8.0.2* on Travis. Any ideas? Could someone reproduce the issue?

**Comment by asr** [comment]
@gallais fixed the issue in 265c916778becd1dac37be4978c2ea1532933130.
